src/ontology/mondo-ingest.Makefile

```diff
@@ -498,6 +499,41 @@ slurp-all-no-updates: $(foreach n,$(ALL_COMPONENT_IDS), slurp-no-updates-$(n))
 .PHONY: slurp-all
 slurp-all: $(foreach n,$(ALL_COMPONENT_IDS), slurp-$(n))
 
+
+#############################
+###### Synchronization ######
+#############################
+.PHONY: sync
+sync: sync-subclassof
```

Comment:
## Pipeline failing due to GARD @matentzn 
@matentzn Can merge or have me fix first.

<details><summary>Details</summary>
<p>

I guess I haven't run via `make` in a few weeks (just Python). Now that I'm running the whole pipeline, it's failing on GARD.

### Possible solutions
I saw the error but it didn't make sense to me yet. I think here are our options
(a) Merge now, fix later
(b) I can remove GARD stuff in a separate PR (most importantly I need to remove `metadata/gard.yml` and then run `make update_repo`, then rebase my PR
(c) I can fix this edge case in my PR so that it successfully runs for GARD, even though we don't need GARD's output

I'm shooting for (b). I need to remove GARD stuff anyway and that should fix the problem.

</p>
</details>

Reply:
Never mind, I fixed via 'c'.